development/extensions/tutorial_testing.rst

```diff
@@ -879,8 +879,8 @@ first file is the Travis CI configuration file, ``.travis.yml``:
       - travis/prepare-phpbb.sh $PHPBB_BRANCH
       - cd ../../phpBB3
       - travis/prepare-extension.sh $EXTNAME $PHPBB_BRANCH
-      - travis/setup-phpbb.sh $DB $TRAVIS_PHP_VERSION
-      - sh -c "if [ '$EPV' == '1' -a '$NOTESTS' == '1' ]; then cd phpBB; composer remove sami/sami --update-with-dependencies --dev --no-interaction; composer require phpbb/epv:dev-master --dev --no-interaction --ignore-platform-reqs; cd ../; fi"
+      - travis/setup-phpbb.sh $DB $TRAVIS_PHP_VERSION $NOTESTS
+      - sh -c "if [ '$EPV' = '1' -a '$NOTESTS' = '1' ]; then cd phpBB; composer remove sami/sami --dev --no-interaction; composer require phpbb/epv:dev-master --dev --no-interaction --ignore-platform-reqs; cd ../; fi"
```

Comment:
They should be `==`

Reply:
Nevermind.

Reply:
never mind

Reply:
I can add it, if it would be better, no big deal.

But I tried it with only one = and that seems to work.